### PR TITLE
New project: SJSU_IT

### PR DESCRIPTION
**Pull request by rynge**

### CODE REVIEW

**Comment by rynge** [comment]
@jstathas

**Comment by jstathas** [comment]
The associated institution San Diego State University is already in the Institution DB. While I’ve heard talk of cleaning up the Project Lists, I’m not aware of a new standard, not has this task been assigned to me.

From: Mats Rynge ***@***.***>
Date: Thursday, May 9, 2024 at 4:39 PM
To: opensciencegrid/topology ***@***.***>
Cc: Stathas, Janet ***@***.***>, Mention ***@***.***>
Subject: Re: [opensciencegrid/topology] New project: SJSU_IT (PR #3873)

@jstathas<https://github.com/jstathas>

**Comment by jstathas** [comment]
Correction – “San Jose State University” is approved.

Janet


From: Stathas, Janet ***@***.***>
Date: Thursday, May 9, 2024 at 4:48 PM
To: opensciencegrid/topology ***@***.***>, opensciencegrid/topology ***@***.***>
Cc: Mention ***@***.***>
Subject: Re: [opensciencegrid/topology] New project: SJSU_IT (PR #3873)
The associated institution San Diego State University is already in the Institution DB. While I’ve heard talk of cleaning up the Project Lists, I’m not aware of a new standard, not has this task been assigned to me.

From: Mats Rynge ***@***.***>
Date: Thursday, May 9, 2024 at 4:39 PM
To: opensciencegrid/topology ***@***.***>
Cc: Stathas, Janet ***@***.***>, Mention ***@***.***>
Subject: Re: [opensciencegrid/topology] New project: SJSU_IT (PR #3873)

@jstathas<https://github.com/jstathas>

File: projects/SJSU_IT.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Department: IT/HPC
+Description: Our goals are to explore and learn about the OSG computing environment
+  so we can help and share the knowledge with SJSU researchers to enable them to utilize
+  OSG HPC for their research projects.
+FieldOfScience: Computer and Information Services
+FieldOfScienceID: '11.0101'
+InstitutionID: Unknown
```
Comment (mwestphall) [inline projects/SJSU_IT.yaml:7]:
```suggestion
InstitutionID: https://osg-htc.org/iid/0r71ijbk5mbz
```

**Review by mwestphall — APPROVED** [review]

**Pull request merged**